setup.cfg

```diff
@@ -0,0 +1,14 @@
+[coverage:run]
+omit = */*/tests/*, */*/settings/*
+
+[isort]
+known_first_party = shared_memory_dict
+atomic = true
+line_length = 79
+multi_line_output = 3
+use_parentheses = true
+include_trailing_comma = True
+force_grid_wrap = 0
+
+[mypy]
+ignore_missing_imports = True
```

Comment:
It would be nice to use `pyproject.toml` to configure this things.

```toml
[tool.isort]
known_first_party = 'shared_memory_dict'
atomic = true
line_length = 79
multi_line_output = 3
use_parentheses = true
include_trailing_comma = true
force_grid_wrap = 0

[tool.coverage.run]
omit = [
    '*/*/tests/*', 
    '*/*/settings/*',
]
```

Reply:
you're right my man, thanks 🤝

Reply:
done ✅

Reply:
🚀